src/content/en/updates/2018/11/webcomponents-v0-deprecations.md

```diff
@@ -0,0 +1,102 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: A round up of the deprecations and removals in Chrome 71 to help you plan.
+
+{# wf_updated_on: 2018-11-26 #}
+{# wf_published_on: 2018-11-13 #}
+{# wf_tags: deprecations,removals,chrome73,webcomponents,origintrials #}
+{# wf_blink_components: Blink>DOM #}
+{# wf_featured_image: /web/updates/images/generic/warning.png #}
+{# wf_featured_snippet: Web Components v0 deprecation and removal in Chrome 73 to help you plan.#}
+
+{% include "web/updates/_shared/see-all-dep-rem.html" %}
+
+# Web Components v0 deprecation {: .page-title }
+
+{% include "web/_shared/contributors/yoichiosato.html" %}
+
+A part of web interoperability for [Web Components v1](/web/updates/2017/01/webcomponents-org),
+we're deprecating Shadow DOM v0, Custom Element v0 and HTML Imports in Chrome 73.
+
+[Intent to Deprecate](https://groups.google.com/a/chromium.org/forum/#!msg/blink-dev/h-JwMiPUnuU/sl79aLoLBQAJ)
+
+
+## Motivation
+
+Shadow DOM v0, Custom Elements v0, and HTML Imports were launched in 2014, but
+they did not get adopted by other browser engines. Instead, Shadow DOM v1,
+Custom Elements v1, and ES modules are widely adopted by various browser engines
+today. Chrome has shipped Shadow DOM V1 / Custom Elements V1 in 2016 and ES
+modules in 2017.
+
+- Shadow DOM v0: [Chromestatus Tracker](https://www.chromestatus.com/feature/4507242028072960) &#124;
+ [Chromium Bug](https://crbug.com/671907)
+- Custom Elements v0: [Chromestatus Tracker](https://www.chromestatus.com/feature/4642138092470272)  &#124;
+ [Chromium Bug](https://crbug.com/660759)
+- HTML Imports: [Chromestatus Tracker](https://www.chromestatus.com/feature/4507242028072960)  &#124;
+[Chromium Bug](https://crbug.com/766694)
+
+### Why at once?
+
+Since then, we also found that in many real-world use cases, Shadow DOM V0,
+Custom Elements V0, HTML Imports were used together, and instead of getting each
+ of them migrated respectively, getting all of them migrated to today’s standards
+ at once, or at least use the polyfill would make it less troublesome, easier to
+ migrate, rather than fighting with every edge case of combination of polyfill
+  and native implementation.
+
+## Migration support
+If you are using this features, please consider migrating to the new API or
+upgrading your Polymer library.
+Use `--disable-blink-features=ShadowDOMV0,CustomElementsV0,HTMLImports` for
+ testing if your site works without those APIs.
+
+## Remove Shadow DOM v0
+`Element.createShadowRoot()` will be removed. Instead we have `Element.attachShadow()`
+to create Shadow Root.
+If your existing components don't use distribution, replacing with
+`Element.attachShadow({mode:"open"})` might work.
```

Comment:
Hmm. These are small differences other than distribution, so I don't recommend to say "might work" here. That could be considered as *too optimistic*, I'm afraid.

Reply:
might work -> we recommend...

Reply:
Changed to just say "not backcompatible".